web/tests/EducationBenchmarking.Web.E2ETests/Features/SchoolHomePage.feature

```diff
@@ -0,0 +1,7 @@
+﻿Feature: School HomePage
+	School homepage feature
+
+Scenario: Go to contact details page
+	Given I am on school homepage for school with urn '139696'
+	When I click on school details in resource section
+	Then I am navigated to school details page for school with urn '139696'
```

Comment:
I'm unclear why asserting the contact page details, etc here is part of this feature? 

Isn't the behaviour here just that you're navigated to the page? Wouldn't the contact details displaying correctly be part of the contact details feature?

Reply:
made the changes and will be documenting this as part of the setting up some standards.